src/components/sbb-breadcrumb-group/sbb-breadcrumb-group.tsx

```diff
@@ -158,91 +158,98 @@ export class SbbBreadcrumbGroup implements ComponentInterface {
       this._state = 'collapsed';
       this._resizeObserver.disconnect();
     }
   }
 
-  private _renderCollapsed(): JSX.Element {
+  private _renderCollapsed(): TemplateResult[] {
     for (let i = 0; i < this._breadcrumbs.length; i++) {
       if (i === 0 || i === this._breadcrumbs.length - 1) {
         this._breadcrumbs[i].setAttribute('slot', `breadcrumb-${i}`);
       } else {
         this._breadcrumbs[i].removeAttribute('slot');
       }
     }
-    const idFirstElement = this._breadcrumbs[0].id ?? `sbb-breadcrumb-0`;
-    const idLastElement =
-      this._breadcrumbs[this._breadcrumbs.length - 1].id ??
-      `sbb-breadcrumb-${this._breadcrumbs.length - 1}`;
 
     return [
-      <li class="sbb-breadcrumb-group__item">
-        <slot
-          name="breadcrumb-0"
-          key={idFirstElement}
-          onSlotchange={(): void => this._readBreadcrumb()}
-        />
-      </li>,
-      <li class="sbb-breadcrumb-group__item" id="sbb-breadcrumb-group-ellipsis">
-        <sbb-icon
-          name="chevron-small-right-small"
-          class="sbb-breadcrumb-group__divider-icon"
-        ></sbb-icon>
-        <button
-          type="button"
-          id="sbb-breadcrumb-ellipsis"
-          aria-label={i18nBreadcrumbEllipsisButtonLabel[this._currentLanguage]}
-          aria-expanded="false"
-          onClick={() => this._expandBreadcrumbs()}
-        >
-          ...
-        </button>
-      </li>,
-      <li class="sbb-breadcrumb-group__item">
-        <sbb-icon
-          name="chevron-small-right-small"
-          class="sbb-breadcrumb-group__divider-icon"
-        ></sbb-icon>
-        <slot
-          name={`breadcrumb-${this._breadcrumbs.length - 1}`}
-          key={idLastElement}
-          onSlotchange={(): void => this._readBreadcrumb()}
-        />
-      </li>,
+      html`
+        <li class="sbb-breadcrumb-group__item">
+          <slot name="breadcrumb-0" @slotchange=${(): void => this._readBreadcrumb()}></slot>
+        </li>
+      `,
+      html`
+        <li class="sbb-breadcrumb-group__item" id="sbb-breadcrumb-group-ellipsis">
+          <sbb-icon
+            name="chevron-small-right-small"
+            class="sbb-breadcrumb-group__divider-icon"
+          ></sbb-icon>
+          <button
+            type="button"
+            id="sbb-breadcrumb-ellipsis"
+            aria-label=${i18nBreadcrumbEllipsisButtonLabel[this._currentLanguage]}
+            aria-expanded="false"
+            @click=${() => this._expandBreadcrumbs()}
+          >
+            ...
+          </button>
+        </li>
+      `,
+      html`
+        <li class="sbb-breadcrumb-group__item">
+          <sbb-icon
+            name="chevron-small-right-small"
+            class="sbb-breadcrumb-group__divider-icon"
+          ></sbb-icon>
+          <slot
+            name=${`breadcrumb-${this._breadcrumbs.length - 1}`}
+            @slotchange=${(): void => this._readBreadcrumb()}
+          ></slot>
+        </li>
```

Comment:
I think it's not necessary to have three different templates in lit. You can have siblings on root level